modules/virt-creating-configmap.adoc

```diff
@@ -1,36 +1,136 @@
 // Module included in the following assemblies:
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
 [id='virt-creating-configmap_{context}']
-= Creating a custom ConfigMap for importing a virtual machine
+= Creating a ConfigMap
 
-If the operating system of the Red Hat Virtualization (RHV) virtual machine does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap to map the RHV operating system to a common {VirtProductName} template. The template is used to create the new virtual machine.
+You can create a ConfigMap to map the Red Hat Virtualization (RHV) virtual machine operating system to an {VirtProductName} template.
 
+ifeval::["{HCOVersion}" == "2.4.0"]
 [NOTE]
 ====
-In the current release, you must create a Windows template and a custom ConfigMap to import a RHV VM with a Windows operating system. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[BZ#1856654] for details.
+If you are importing a Windows virtual machine, you must create a Windows template in addition to a ConfigMap. link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[(*BZ#1856654*)]
 ====
+endif::[]
 
 .Procedure
 
-. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` as in the following example:
+. In a web browser, identify the REST API name of the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>`. The operating system name appears in the `<os>` section of the XML output, as shown in the following example:
 +
 [source,xml]
 ----
+...
 <os>
 ...
-<type>windows_2016x64</type>
+<type>rhel_8x64</type>
 </os>
 ----
 
-. Obtain a list of the common {VirtProductName} operating system templates by running the following command on your {product-title} cluster:
+. Obtain a list of the available {VirtProductName} templates:
 +
 ----
 $ oc get templates -n openshift --show-labels | tr ',' '\n' | grep os.template.kubevirt.io | sed -r 's#os.template.kubevirt.io/(.*)=.*#\1#g' | sort -u
 ----
 +
-If the RHV VM operating system does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap.
+The output is similar to the following example:
++
+----
+fedora31
+fedora32
+...
+rhel8.1
+rhel8.2
+...
+----
+
+. If a template corresponding to the RHV VM operating system does not appear in the list of available templates, create a template with the {VirtProductName} web console.
+
+. Create a ConfigMap to map the RHV VM operating system to the template:
++
+[source,yaml]
```

Comment:
I would suggest generally splitting this into two parts (creating a yaml file and running the oc create -f command). I think it's generally a better user experience to split it up if they are likely to edit the yaml at all. 

One newer consideration is that this is technically a terminal command -- does it get [source,terminal] highlighting even though it contains a yaml? cc @vikram-redhat

Reply:
@ousleyp This whole section is a workaround for 2.4.0. It will be removed for 2.4.1 in a couple weeks. I don't want to invest too much time in reorganizing it.